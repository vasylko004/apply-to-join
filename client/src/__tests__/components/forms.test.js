import React from 'react';
import renderer from 'react-test-renderer';
import ApplyToJoinForm  from "../../components/forms/apply-to-join"

test("render apply to join", async ()=>{
    const tree = renderer
        .create(<ApplyToJoinForm />)
        .toJSON();
    expect(tree).toMatchSnapshot();
} )