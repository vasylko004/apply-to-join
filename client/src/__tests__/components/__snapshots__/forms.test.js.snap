// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` apply to join renders correctly 1`] = `
<div>
  <form
    action="#"
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Name
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="name"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Name"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Email
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="email"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Email"
              required={true}
              type="email"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Phone Number
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="phone"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Phone Number"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-7"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Address
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="address"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Address"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-5"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Zip Code
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="zipCode"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Zip Code"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <section
          className="dropzone-section"
        >
          <div
            onBlur={[Function]}
            onClick={[Function]}
            onDragEnter={[Function]}
            onDragLeave={[Function]}
            onDragOver={[Function]}
            onDrop={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            tabIndex={0}
          >
            <input
              autoComplete="off"
              multiple={true}
              onChange={[Function]}
              onClick={[Function]}
              style={
                Object {
                  "display": "none",
                }
              }
              tabIndex={-1}
              type="file"
            />
            <p>
              Drag 'n' drop some files here, or click to select files
            </p>
          </div>
        </section>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="submit"
        >
          <span
            className="MuiButton-label"
          >
             Submit 
          </span>
        </button>
      </div>
    </div>
  </form>
</div>
`;

exports[`render apply to join 1`] = `
<div>
  <form
    action="#"
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Name
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="name"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Name"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Email
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="email"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Email"
              required={true}
              type="email"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Phone Number
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="phone"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Phone Number"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-7"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Address
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="address"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Address"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-5"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
            data-shrink={false}
          >
            Zip Code
            <span
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              name="zipCode"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Zip Code"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <section
          className="dropzone-section"
        >
          <div
            onBlur={[Function]}
            onClick={[Function]}
            onDragEnter={[Function]}
            onDragLeave={[Function]}
            onDragOver={[Function]}
            onDrop={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            tabIndex={0}
          >
            <input
              autoComplete="off"
              multiple={true}
              onChange={[Function]}
              onClick={[Function]}
              style={
                Object {
                  "display": "none",
                }
              }
              tabIndex={-1}
              type="file"
            />
            <p>
              Drag 'n' drop some files here, or click to select files
            </p>
          </div>
        </section>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="submit"
        >
          <span
            className="MuiButton-label"
          >
             Submit 
          </span>
        </button>
      </div>
    </div>
  </form>
</div>
`;
